evennia/contrib/tests.py

```diff
@@ -688,8 +688,8 @@ def test_mail(self):
                   "You have received a new @mail from TestAccount2(account 2)|You sent your message.", caller=self.account2)
         self.call(mail.CmdMail(), "TestAccount=Message 1", "You sent your message.", caller=self.account2)
         self.call(mail.CmdMail(), "TestAccount=Message 2", "You sent your message.", caller=self.account2)
-        self.call(mail.CmdMail(), "", "| ID:   From:            Subject:", caller=self.account)
-        self.call(mail.CmdMail(), "2", "From: TestAccount2", caller=self.account)
+        self.call(mail.CmdMail(), "", "------------------------------------------------------------------------------| ID:   From:            Subject:", caller=self.account)
```

Comment:
It think this may be wrong. We *don't* want things like this to appear in the test suite. The way to correct this is rather to limit so that you don't allow more than two `-` in a row.

Reply:
Hmm, good point. Any idea why `--+` isn't matching that in the regex, while `--*` did? My understanding is the former would match any string of two or more dashes, while the latter would be one or more, so I'm not quite getting why it doesn't match a strong series of them.

Reply:
It's actually surprising a string like `"------------------------------"` would not be stripped with your change, it seems from the reges this new form should catch and remove that string too.

Reply:
Yeah, I just copied the regex into shell and was testing it against the test string, and `_RE.sub` is eating the test string, so I'm not sure what's going on there. How strange. I'll keep poking at it.